### PR TITLE
Add DB changes for multiple bundle stores

### PR DESCRIPTION
**Pull request by adiprerepa**
for multi bundle store

design doc:https://docs.google.com/document/d/1TOsO4gnxUm1XMmlGUb7BJ4Z6y_olFbv9APbx0-IEy6M/edit#heading=h.kh5hv3iuzlpg

### CODE REVIEW

File: codalab/model/tables.py [hunk 1]
```diff
@@ -93,6 +93,73 @@
...
+    # URL that uniquely identifies the bundle store.
+    Column('url', String(255), nullable=True),
+    
+    # Authentication / password to access the bundle store.
+    Column('authentication', String(255), nullable=True),
+
+    # Name of environment variable through which the authentication variable from above can be passed to codalab-service to access the bundle store.
+    Column('authentication_env', String(255), nullable=True),
+
+    UniqueConstraint('uuid', 'owner_id', name='uix_1'),
```
Comment (epicfaace) [inline codalab/model/tables.py]: I think we discussed that we would make it unique on uuid only (instead of `(uuid, owner_id)`)

**Review by epicfaace — CHANGES_REQUESTED** [review]

File: codalab/model/tables.py (hunk 1 above)
Comment (adiprerepa) [reply codalab/model/tables.py]: done

File: codalab/model/tables.py [hunk 2]
```diff
@@ -93,6 +93,73 @@
...
+        autoincrement=True,
+    ),
+    Column('uuid', String(63), nullable=False),
+
+    # Usually root. For BYO bundle stores (not yet supported),
+    # it is the user who created the bundle store.
+    Column('owner_id', String(255), nullable=True),
+
+    # Name used to refer to this bundle store from the CLI.
+    Column('name', String(255), nullable=True),
```
Comment (epicfaace) [inline codalab/model/tables.py]: Let's create an index on name and on owner, just like we do for worksheets -- see https://github.com/codalab/codalab-worksheets/blob/7544ef4ace20722881d1d25f7c864341a97617bd/codalab/model/tables.py#L190

**Review by epicfaace — CHANGES_REQUESTED** [review]

File: codalab/model/tables.py (hunk 2 above)
Comment (adiprerepa) [reply codalab/model/tables.py]: done

**Review by epicfaace — APPROVED** [review]

File: codalab/model/tables.py [hunk 3]
```diff
@@ -93,6 +93,64 @@
...
+        'id',
+        BigInteger().with_variant(Integer, "sqlite"),
+        primary_key=True,
+        nullable=False,
+        autoincrement=True,
+    ),
+    Column('uuid', String(63), nullable=False),
+    # Usually root. For BYO bundle stores (not yet supported),
+    # it is the user who created the bundle store.
+    Column('owner_id', String(255), nullable=True),
```
Comment (epicfaace) [inline codalab/model/tables.py:110]: Make this nullable False

File: codalab/model/tables.py [hunk 4]
```diff
@@ -93,6 +93,64 @@
...
+    # Name used to refer to this bundle store from the CLI.
```
Comment (epicfaace) [inline codalab/model/tables.py:111]: Make `name` nullable False

Comment (epicfaace) [reply codalab/model/tables.py:111]: We should do this to correspond with the worksheet table. Can you also check all other columns with worksheet to ensure they correspond in terms of things being nullable, etc.?

**Review by epicfaace — APPROVED** [review]
Feel free to merge after addressing these last comments

File: codalab/model/tables.py (hunk 4 above)
Comment (adiprerepa) [reply codalab/model/tables.py:111]: owner_id on bundle worksheet is nullable=True

Comment (adiprerepa) [reply codalab/model/tables.py:111]: matched everything with worksheet

Comment (epicfaace) [reply codalab/model/tables.py:111]: Oh, I guess that's kind of strange (having it nullable = True) but let's just keep it that way to be consistent

**Review by epicfaace — APPROVED** [review]

**Pull request merged**